show/main.py

```diff
@@ -172,8 +171,8 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
-# 'summary' subcommand ("show interfaces summary")
-@interfaces.command()
+# 'summary' subcommand ("show interfaces summary") -- called if no subcommands are passed
+@interfaces.command(default=True)
```

Comment:
I am not a fan of the `DefaultGroup` we have implemented using Click, because when a group has a default command, it does not allow for using Tab to show available subcommands.

We either need to eliminate default commands for groups altogether, or try to fix our DefaultGroup to allow the Tab functionality to work.

Reply:
For the time being, will get rid of this 'default' behavior. Agree that 'tabs' is a nice-to-have feature.